samza-core/src/test/java/org/apache/samza/config/TestStorageConfig.java

```diff
@@ -155,7 +154,7 @@ public void testGetStorageMsgSerde() {
   }
 
   @Test
-  public void testGetChangelogSystem() {
+  public void getChangelogStream() {
```

Comment:
There is already a test called `testGetChangelogStream`. Should this be merged with that?

Reply:
done